src/errorpage.h

```diff
@@ -351,5 +351,6 @@ class TemplateFile
  */
 bool strHdrAcptLangGetItem(const String &hdr, char *lang, int langLen, size_t &pos);
 
+std::ostream &operator <<(std::ostream &, ErrorState *);
```

Comment:
Rule of thumb: If you can make it `const`, make it `const`.

```suggestion
std::ostream &operator <<(std::ostream &, const ErrorState *);
```